site/source/docs/site/about.rst

```diff
@@ -56,7 +56,7 @@ Ubuntu
 
 The version of Sphinx on Ubuntu package repository (apt-get) fails when building the site. This is an early version (1.1.3), which appears to be dependent on an old version of the Jinja templating library.
 
-The workaround is to use the *Python package installer* (pip) to get version 1.7.8, and then run an upgrade (note, you may have to uninstall Sphinx first): ::
+The workaround is to use the *Python package installer* (pip) to get version 2.4.4, and then run an upgrade (note, you may have to uninstall Sphinx first): ::
 
   pip install sphinx==2.4.4
```

Comment:
Can you instead replace this entire sections with something like: "The site requires a specific version sphins to be installed.  Run the following command to ensure you have the correct version installed:  ..."  then list the `pip install -r requirements-dev.txt` command?

Reply:
Sounds good. Then given that we require 2.4.4 specifically, do we need this Ubuntu-specific section at all? Apparently you would need to run `pip install -r requirements-dev.txt` whether you are on Ubuntu or not, right?

Also, I guess we need to fix the sentence at the top of this page, which says this site is built with 1.2.2:
> The site is built using [Sphinx](http://sphinx-doc.org/latest/index.html) (1.2.2), the open source tool used to create the official Python documentation and many other sites. This is a very mature and stable tool, and was selected for, among other reasons, its support for defining API items and linking to them from code.

Reply:
Non need to mention ubuntu no.. the requirements are the same on every platform, as is the command used to install

Reply:
Done